src/helpers/manage-merge-queue.ts

```diff
@@ -92,6 +98,15 @@ const addPrToQueue = async (pullRequest: PullRequest, queuePosition: number) =>
     enableAutoMerge(pullRequest.node_id)
   ]);
 
+const addPrToQueue = async (queuePosition: number) =>
+  Promise.resolve(
+    octokit.issues.addLabels({
```

Comment:
not a big deal but if this code block is duplicated it might make sense to abstract it, or rather just call it in `addPrToQueueWithAutoMerge`

Reply:
Made update.